cmd/release/eval/eval.go

```diff
@@ -99,7 +101,9 @@ func (o *options) run() error {
 
 // Print prints the successful outcome of the cmd
 func (o *options) Print() {
-	color.Green("Release entry successfully logged!")
-
-	// fmt.Println("\n" + o.Release.String())
+	if o.bCtx.CLIConfig.Color {
+		color.Green("Release entry successfully logged!")
+	} else {
+		fmt.Println("Release entry successfully logged!")
+	}
```

Comment:
I'd avoid duplicating the string literal (unless Go recognises that it's the same string, but I don't think it's that smart). 

```go
const s string  = "Release entry successfully logged!"
if ... {
  color.Green(s)
else {
  fmt.Println(s)
}
```